qcodes/instrument/parameter.py

```diff
@@ -1295,6 +1295,14 @@ def __init__(
                 '',
                 self.__doc__))
 
+    @property
+    def unit(self) -> Optional[str]:
```

Comment:
actually unit cannot be None, see line 1270
```suggestion
    def unit(self) -> str:
```